### PR TITLE
Create a FAQ section

### PR DESCRIPTION
**Pull request by systemcatch**
There's some recurring questions that people tend to ask about the electricitymap so I though it would be useful to start an FAQ section.  I've added some of things I think might be useful but will add anything else that @corradio  @jarek @brunolajoie think is important.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,20 @@ You can also see a list of missing data displayed as warnings in the developer c
 
 Check the [contributing](#contribute) section for more details.
 
+## Frequently asked questions
+
+**How do you define real time data?**
```
Comment (corradio) [inline README.md]: Spelling should be "real-time" (with a dash) in all the text I think.

File: README.md [hunk 2]
```diff
@@ -19,6 +19,20 @@ You can also see a list of missing data displayed as warnings in the developer c
 
 Check the [contributing](#contribute) section for more details.
 
+## Frequently asked questions
+
+**How do you define real time data?**  
+Real time data is defined as a data source that updates every 2 hours or better.  It should provide a breakdown by generation type.  Often fossil fuel generation (coal/gas/oil) is combined under a single heading like 'thermal' or 'conventional', this is not a problem.
+
+**Is non real time data useful?**  
+While the map relies on having real-time data to work it's still useful to collect data from days/months past. This older data can used to show past emissions and build up a better dataset. So if there's a good data source that lags several days behind you can still build a parser for it.
```
Comment (corradio) [inline README.md]: Maybe insist on the fact it needs to be hourly at least?

File: README.md [hunk 3]
```diff
@@ -19,6 +19,20 @@ You can also see a list of missing data displayed as warnings in the developer c
 
 Check the [contributing](#contribute) section for more details.
 
+## Frequently asked questions
+
+**How do you define real time data?**  
+Real time data is defined as a data source that updates every 2 hours or better.  It should provide a breakdown by generation type.  Often fossil fuel generation (coal/gas/oil) is combined under a single heading like 'thermal' or 'conventional', this is not a problem.
```
Comment (corradio) [inline README.md]: Maybe say that it is defined as a data source with an hourly (or better) frequency, delayed by less than 2hrs.

File: README.md [hunk 4]
```diff
@@ -19,6 +19,20 @@ You can also see a list of missing data displayed as warnings in the developer c
...
+
+**Is non real time data useful?**
```
Comment (corradio) [inline README.md]: Maybe focus the title on the question of whether or not **delayed** data is still useful (instead of real-time)

**Comment by systemcatch** [comment]
Sure I'll make the changes you've suggested.  Any other questions you want added?

**Comment by corradio** [comment]
Maybe we could add a section about why we use lifecycle emissions, ref #960 ?
But I can add that later.

**Pull request merged**